src/features/checkout/screen/formInput.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Controller } from 'react-hook-form';
+import { StyleSheet, Text, TextInput } from 'react-native';
+
+const FormInput = ({ control, name, ...otherProps }) => {
```

Comment:
I think you should destruct every value you use directly here, like the label.

```suggestion
const FormInput = ({ control, name, label, ...otherProps }) => {
```